### PR TITLE
added autogen of terraform docs

### PR DESCRIPTION
**Pull request by batthebee**
solves #203

### CODE REVIEW

**Comment by batthebee** [comment]
@strowi can you take a look on this?

File: README.md [hunk 1]
```diff
@@ -318,7 +428,7 @@ There is also a branch where openSUSE MicroOS came preinstalled with the k3s RPM
...
 
-🌱 This project currently installs openSUSE MicroOS via the Hetzner rescue mode, making things a few minutes slower. If you could **take a few minutes to send a support request to Hetzner, asking them to please add openSUSE MicroOS as a default image**, not just an ISO. The more requests they receive, the likelier they are to add support for it, and if they do, that will cut the deployment time by half. The official link to openSUSE MicroOS is <https://get.opensuse.org/microos>, and their `OpenStack Cloud` image has full support for Cloud-init, which would probably suit the Hetzner Ops team!
+🌱 This project currently installs openSUSE MicroOS via the Hetzner rescue mode, making things a few minutes slower. If you could **take a few minutes to send a support request to Hetzne, asking them to please add openSUSE MicroOS as a default image**, not just an ISO. The more requests they receive, the likelier they are to add support for it, and if they do, that will cut the deployment time by half. The official link to openSUSE MicroOS is <https://get.opensuse.org/microos>, and their `OpenStack Cloud` image has full support for Cloud-init, which would probably suit the Hetzner Ops team!
```
Comment (nupplaphil) [inline README.md:431]:
```suggestion
🌱 This project currently installs openSUSE MicroOS via the Hetzner rescue mode, making things a few minutes slower. If you could **take a few minutes to send a support request to Hetzner, asking them to please add openSUSE MicroOS as a default image**, not just an ISO. The more requests they receive, the likelier they are to add support for it, and if they do, that will cut the deployment time by half. The official link to openSUSE MicroOS is <https://get.opensuse.org/microos>, and their `OpenStack Cloud` image has full support for Cloud-init, which would probably suit the Hetzner Ops team!
```

**Review by nupplaphil — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -276,6 +276,8 @@ Same goes for all add-ons, like Longhorn, Cert-manager, and Traefik.
 
 </details>
 
+## Reference
```
Comment (nupplaphil) [inline README.md:279]: This seems redundant/unnecessary

**Review by nupplaphil — CHANGES_REQUESTED** [review]

**Comment by strowi** [comment]
> @strowi can you take a look on this?

Perfect @batthebee should be very helpful to have the variables autogenerated.  ;)

**Review by strowi — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (mysticaltech) [reply README.md:431]: Indeed, good catch, @nupplaphil! 🥳

File: README.md (hunk 2 above)
Comment (mysticaltech) [reply README.md:279]: I see what you mean; I would be more for saving this in a new md file.

**Comment by mysticaltech** [comment]
@batthebee Can we instead save this in a new file? We already have a docs folder with SSH.md in it, maybe we could add Terraform.md, and link to it from the main README.md. I would prefer it that way! Especially if it can be auto-regenerated, it would be even more awesome!

**Comment by batthebee** [comment]
@mysticaltech do you have a suggestion in which section i should insert the link in the README.md?

**Comment by mysticaltech** [comment]
@batthebee At the end of "creating your kube.tf" would be ideal! 🙏

**Comment by mysticaltech** [comment]
@batthebee If you can finish fast, it would be an awesome addition to our upcoming 1.3.0 release that is due when we merge your other PR! 🤞

File: README.md (hunk 1 above)
Comment (batthebee) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (batthebee) [reply README.md]: Done

**Review by nupplaphil — APPROVED** [review]

**Review by mysticaltech — APPROVED** [review]
Wonderful!

**Pull request merged**